cmd/loki/loki-local-config.yaml

```diff
@@ -18,11 +18,11 @@ ingester:
 schema_config:
```

Comment:
Let's revert that file. You can add a new file next time and add it to your global git gitignore.